### PR TITLE
feat: merge ProgressCircle into ActivityIndicator

### PR DESCRIPTION
**Pull request by chambo-e**
## Type

- Feature

#### The following changes where made:

Merged ProgressCircle into ActivityIndicator while keeping 99% of ProgressCircle code

Changes:
- default size is not 10 anymore but 40
- to have ActivityIndicator behaviour we need to enable `active` prop
- ActivityIndicator does not accept xstyled props anymore

## Relevant logs and/or screenshots

https://user-images.githubusercontent.com/2802867/120491156-4a6dad80-c3b9-11eb-85cc-3d6057b13602.mov

### CODE REVIEW

**Review by matthprost — CHANGES_REQUESTED** [review]
Its a little confusing ahah

Screenshot 2021-06-04 at 11 12 12

**Review by vincentaudebert — APPROVED** [review]
approved after Matthias comments fixed.

File: src/components/ActivityIndicator/__stories__/index.stories.mdx [hunk 1]
```diff
@@ -5,18 +5,42 @@ import {
...
 
 # ActivityIndicator
 
-<Description>Component that indicate a loading period.</Description>
+<Description>
+  A component representing percentage as a circle which can be used to show
+  activity
+</Description>
+
+## Basic
```
Comment (DorianMaliszewski) [inline src/components/ActivityIndicator/__stories__/index.stories.mdx:20]: We dropped Basic title with Documentation refacto

[the remark this answers is not in the document]
Comment (chambo-e) [reply src/components/ActivityIndicator/__stories__/index.stories.mdx:20]: Corrected :)

Comment (chambo-e) [reply src/components/ActivityIndicator/__stories__/index.stories.mdx:20]:
```suggestion
```

**Review by DorianMaliszewski — APPROVED** [review]

**Pull request merged**